### PR TITLE
CAMS-294 Updates to data import script

### PR DESCRIPTION
**Pull request by magile32**
# Purpose

Import anonymized data from USTP into Flexion environment for multiple tables.

# Major Changes

- Added new script to help with setting developer roles to access Cosmos db account
- Fixed existing issues with data import script
- Enhancement to data import script to enable loading multiple data files in target directory
- Removed dataloader java project
- Removed references to java formatter 

# Testing/Validation

- Verify data import script operate as expected on multiple data files
- Confirm CAMS application works with new anonymized data (with exception noted below)

# Notes

- Anonymized data had incorrect format for court_id for select tables. Value needed to be padded with zeros up to 4 integers. See `ops/scripts/utility/fix-import-data.sql`
- Also, we noticed some possibly invalid records (4) in `ao_alias.csv ` containing a pipe character in a column reserved for a person's name which threw some errors.

### CODE REVIEW

**Comment by jharcey-flexion** [comment]
> # Notes
> * Anonymized data had incorrect format for case_id for select tables. Value needed to be padded with zeros up to 4 integers. See `ops/scripts/utility/fix-import-data.sql`
> * Also, we noticed some possibly invalid records (4) in `ao_alias.csv ` containing a pipe character in a column reserved for a person's name which threw some errors.

Wasn't the incorrectly formatted data the COURT_ID?

@governmentSponsored could you also add the second note to your process?

**Comment by magile32** [comment]
> > # Notes
> > 
> > * Anonymized data had incorrect format for case_id for select tables. Value needed to be padded with zeros up to 4 integers. See `ops/scripts/utility/fix-import-data.sql`
> > * Also, we noticed some possibly invalid records (4) in `ao_alias.csv ` containing a pipe character in a column reserved for a person's name which threw some errors.
> 
> Wasn't the incorrectly formatted data the COURT_ID?
> 
> @governmentSponsored could you also add the second note to your process?

Yes. Updated the PR notes

**Review by fmaddenflx — APPROVED** [review]
Looks okay.  I didn't double check all your regular expressions, but at first glance they look okay.

File: ops/scripts/utility/fix-import-data.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- Fix anonymized data set for the following table to ensure zero padding of COURT_ID
+-- Execute this after a data import to ensure correct formatting
+update AO_PY set COURT_ID='0'+COURT_ID WHERE LEN(COURT_ID)<4;
```
[the remark this answers is not in the document]
Comment (magile32) [reply ops/scripts/utility/fix-import-data.sql]: Updated sql statement to account for a court id with 2 significant characters.

**Pull request merged**